packages/core/src/workflow/create-tx/CreateBitcoinTx.ts

```diff
@@ -21,80 +20,75 @@ export interface Output {
 }
 
 export interface TxMetric {
-  /**
-   * https://en.bitcoin.it/wiki/Weight_units
-   *
-   * @param inputs
-   * @param outputs
-   */
+  // https://en.bitcoin.it/wiki/Weight_units
   weightOf(inputs: BalanceUtxo[], outputs: Output[]): number;
 }
 
 class AverageTxMetric implements TxMetric {
-
   weightOf(inputs: BalanceUtxo[], outputs: Output[]): number {
-    //TODO incorrect for many scenarios, based on numbers from https://en.bitcoin.it/wiki/Weight_units "Detailed Example" taken as is
-    return 22 +
-      (128 + 16 + 4 + 92) * inputs.length +
-      16 + 4 +
-      (32 + 4 + 100 + 1 + 1 + 72 + 1 + 33 + 16) * outputs.length;
+    /*
+     * TODO
+     *   Incorrect for many scenarios, based on numbers from https://en.bitcoin.it/wiki/Weight_units
+     *   "Detailed Example" taken as is
+     */
+    return (
+      22 + (128 + 16 + 4 + 92) * inputs.length + 16 + 4 + (32 + 4 + 100 + 1 + 1 + 72 + 1 + 33 + 16) * outputs.length
+    );
   }
-
 }
 
+// https://bitcoinfees.net/, https://www.buybitcoinworldwide.com/fee-calculator
+const DEFAULT_VKB_FEE = 1024;
+
 // see https://en.bitcoin.it/wiki/Weight_units
 export function convertWUToVB(wu: number): number {
   return wu / 4;
 }
 
-// https://bitcoinfees.net/, https://www.buybitcoinworldwide.com/fee-calculator
-const DEFAULT_VKB_FEE = 1024;
+export class CreateBitcoinTx {
```

Comment:
Can you please tell why `<A extends BigAmount>` was remove and replaced with just `BigAmount` in the code?